### PR TITLE
Using sponsor id to filter project list

### PR DESCRIPTION
**Pull request by PierreOucif**

### CODE REVIEW

**Review by AnthonyBuisset — APPROVED** [review]

**Review by AnthonyBuisset — COMMENTED** [review]
Nice work

**Comment by AnthonyBuisset** [comment]
Are you sure this is needed ?

File: domain/src/main/java/onlydust/com/marketplace/api/domain/port/output/ProjectStoragePort.java [hunk 1]
```diff
@@ -17,12 +17,12 @@ public interface ProjectStoragePort {
 
     ProjectDetailsView getBySlug(String slug);
 
-    Page<ProjectCardView> findByTechnologiesSponsorsUserIdSearchSortBy(List<String> technologies, List<String> sponsors,
```
Comment (AnthonyBuisset) [inline domain/src/main/java/onlydust/com/marketplace/api/domain/port/output/ProjectStoragePort.java:20]: Good idea !

Comment (AnthonyBuisset) [reply domain/src/main/java/onlydust/com/marketplace/api/domain/port/output/ProjectStoragePort.java:20]: Indeed !

Comment (PierreOucif) [reply domain/src/main/java/onlydust/com/marketplace/api/domain/port/output/ProjectStoragePort.java:20]: Thx !!

**Pull request merged**